tools/bazelw

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -euo pipefail
+
+readonly bazel_version="0.19.0"
```

Comment:
What's the problem with modifying this when the version changes? Since it isn't generated. If we did that we'd also have to do that with the sha's and the platform conditional

Reply:
Allows easier upgrades of the script, as discussed locally it's a reference rather than an implementation, so I guess it's fine